tfe/resource_tfe_organization_test.go

```diff
@@ -0,0 +1,207 @@
+package tfe
+
+import (
+	"fmt"
+	"testing"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccTFEOrganization_basic(t *testing.T) {
+	org := &tfe.Organization{}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckTFEOrganizationDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccTFEOrganization_basic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckTFEOrganizationExists(
+						"tfe_organization.foobar", org),
+					testAccCheckTFEOrganizationAttributes(org),
```

Comment:
I'd argue we don't need this since we're validating these below

Reply:
I disagree. This test is checking if the values were written correctly to the state file. The other test is checking if the returned object contains the expected values.

Reply:
You can also validate this via `ImportStateVerify` testing, since it seems this resource is easily able to be setup for resource import.

In the schema:

```go
Importer: &schema.ResourceImporter{
  State: schema.ImportStatePassthrough,
},
```

As a new `TestStep`:

```go
{
  ResourceName: "tfe_organization.foobar",
  ImportState: true,
  ImportStateVerify: true,
},
```

Documented at the bottom of the resource documentation:

``````
## Import

`aws_acmpca_certificate_authority` can be imported by using the certificate authority Amazon Resource Name (ARN), e.g.

```
$ terraform import tfe_organization.example OrganizationName
```
``````

Reply:
If you don't mind I would like to postpone adding import (and datasources) the a next PR. This is big enough as it is right now :wink:

But good point and will certainly think about this in the next iteration 👍